Tasks/Common/packaging-common/Tests/npmL0.ts

```diff
@@ -127,7 +127,7 @@ export function npmcommon() {
     });
```

Comment:
The changes to this file are for Mocha 3.0 support (Error: Resolution method is overspecified. Specify a callback *or* return a Promise; not both.)